mercadopago/px-ios
# (Fixed)[PXN-4381] - Fixing sonic branding

## What have changed?
Fixing sonic branding by using the appropiated status

## How to test:
Navigate through the checkout with any kind of payment status to get a congrats screen. A sound should be played accordingly to the payment status.

## Prints for layout changes:
`N/A`

## Extras:
`N/A`
